### PR TITLE
Update Hyrax metadata fails of the list of the granules won't contain a metadata file

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
**Summary:** Summary of changes
Update Hyrax metadata fails of the list of the granules won't contain a metadata file 

Addresses [CUMULUS-2953: Add skipMetaCheck to HyraxMetadataUpdate](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2953)
## Changes

* Adding skipMetaCheck to hyrax-metadata-updates lambda function
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: tasks/hyrax-metadata-updates/index.js [hunk 1]
```diff
@@ -308,11 +308,21 @@ const updateGranule = (config) => async (granule) => {
   addEtagsToFileObjects(granule, etags);
   // Read in the metadata file
   const metadataFile = granule.files.find(isCMRFile);
-  // If there is no metadata file, error out.
+  // If there is no metadata file.
   if (metadataFile === undefined) {
+    // Skip metadata check if skipMetaCheck set to true
```
Comment (Jkovarik) [inline tasks/hyrax-metadata-updates/index.js:313]: 👍🏻  this seems reasonable, but it needs a unit test addition  (e.g. https://github.com/nasa/cumulus/blob/master/tasks/hyrax-metadata-updates/tests/test-integration-index.js)

**Comment by Jkovarik** [comment]
@amarouane-ABDELHAK  are you willing/able to add a unit test, or did you have other concerns?  Would be happy to work with you on this if needed.

**Comment by amarouane-ABDELHAK** [comment]
Just we need to allocate a time to add unittests for this PR. Let me get back to you

**Comment by nemreid** [comment]
Closed as same functionality was merged in https://github.com/nasa/cumulus/pull/3014

**Pull request closed without merging**